paddle/framework/attr_type.proto

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
I am not sure.
But [tensorflow](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/core/framework)  make protobuf files into sub-directories. Since we are using google code style now, it seems that we should put our protobuf files in sub-directories.

Another advantage that make protobuf file in sub-directories is it make include line pretty like below.

```cpp
#include <paddle/framework/op_proto.pb.h>
```